lisa/sut_orchestrator/azure/features.py

```diff
@@ -77,6 +77,12 @@ def _initialize_information(self, node: Node) -> None:
 
 
 class StartStop(AzureFeatureMixin, features.StartStop):
+    @classmethod
+    def check_supported(
```

Comment:
Can we rename it something else? `check_supported` seems like a method name that should return a bool. Maybe something like `get_feature_if_supported`?

Reply:
good suggestion, will rename to `create_setting`